server/src/schedulers/stellarobject_timers.rs

```diff
@@ -1,18 +1,11 @@
 use std::{f32::consts::PI, time::Duration};
 use glam::Vec2;
-use log::info;
 use spacetimedb::{ReducerContext, Table};
+use spacetimedsl::{dsl};
 
-use crate::types::{common::global_config, stellarobjects::*};
-
-// // Intentionally private
-// #[spacetimedb::table(name = update_config)]
-// pub struct UpdateConfig {
-//     #[unique]
-//     id: u32,
-//     value: i32,
-// }
+use crate::types::{common::{are_there_active_players, try_server_only}, stellarobjects::*};
 
+//#[dsl(plural_name = update_transforms_timers)]
```

Comment:
I would call them `update_sobj_transform_timers` instead (just add a `s` at the end). I unfortunately can't provide a default because of regular vs. irregular nouns in the english language where some end with `s` in plural form, some with `es`, some with `ies` instead of `y`, some use the same word or have completely different plural forms.